site/package.json

```diff
@@ -30,12 +30,28 @@
     "gulp-rev-delete-original": "^0.2.3",
     "gulp-stylus": "^2.7.0",
     "gulp-uglify-es": "^1.0.4",
-    "gulp-webserver": "^0.9.1"
+    "gulp-webserver": "^0.9.1",
+    "husky": "^1.2.0",
+    "lint-staged": "^8.1.0",
+    "xo": "^0.23.0"
   },
-  "author": "Swapnil Agarwal",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/swapagarwal/swag-for-dev/issues"
+  "scripts": {
+    "lint": "xo '**/*.js'",
+    "lint:fix": "xo --fix '**/*.js'",
+    "build": "gulp build"
+  },
+  "xo": {
+    "space": true
```

Comment:
Is there a reason for this override?

Reply:
Also, to expand on this... either way we should use 4 spaces to maintain legibility. With Promises and async, nested callbacks are virtually nonexistant which (afaik) was one of the big reasons for using 2 spaces

Reply:
moved to 4 spaces